exercises/crypto-square/src/example.c

```diff
@@ -0,0 +1,92 @@
+#include "crypto_square.h"
+#include <stdlib.h>
+#include <ctype.h>
+#include <stdio.h>
+#include <math.h>
+
+/* mstrlen: get the length of input counting only alnums */
+static int mstrlen(const char *input)
+{
+   int i = 0;
+   while (*input) {
```

Comment:
You have two of these loops along input. I think you should be explicit about what you're testing against just to be a bit clearer. `*input != '\0'`

Additionally, I personally feel that writing these as a for loop is clearer. No need to change that if you don't want to though.

```
for (; *input != '\0'; input++) {
  if (isalnum(*input)) {
    i++;
  }
}
```